### PR TITLE
Paper: High-performance operator evaluations with ease of use: libCEED's Python interface

### PR DESCRIPTION
**Pull request by valeriabarra**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

### CODE REVIEW

File: make_paper.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ if [[ ! -d $DIR ]]; then
   exit -1
 fi
 
-python publisher/build_paper.py $DIR
+python3 publisher/build_paper.py $DIR
```
Comment (deniederhut) [inline make_paper.sh]: We prefer that paper PRs not modify any files outside of their own directory. Would you mind reverting this change?

Comment (valeriabarra) [reply make_paper.sh]: Absolutely, sorry. I meant to do that already but it must have slipped.

Comment (deniederhut) [reply make_paper.sh]: No worries - thanks for making the change!

**Comment by vahtras** [comment]
Hi, this is to introduce myself as a reviewer. Not done yet, but looks very good so far

**Comment by valeriabarra** [comment]
Thanks for the introduction @vahtras  and thanks for your review!

**Comment by vahtras** [comment]
Regarding the code snippets, we now have a mixture of bash and Python with the same font/notation. This is unfortunate. I strongly recommend the established practice of writing out the prompts in both cases, e.g.

    $ pip install libceed

vs

    >>> import libceed

i.e. all Python examples written as interactive sessions. If an output is expected from a Python instruction (e.g. print) it should be written out, such that it can be verified with doctest

**Comment by valeriabarra** [comment]
Sure, thanks for pointing that out. I will take care of it.

**Comment by valeriabarra** [comment]
Thanks for the feedback. 
For now I have added the prompts in the snippets only. I will add the outputs and try to verify with doctest

**Comment by deniederhut** [comment]
Hey @valeriabarra ! Thanks to some awesome behind the scenes work by @stargaser, we now have experimental support for publishing papers with ORCIDs. If you have one and would like to add it to your paper, it's as simple as adding an `:orcid:` tag under your name in the paper header. [You can see an example of how to do this in the `00_vanderwalt` example paper](https://raw.githubusercontent.com/scipy-conference/scipy_proceedings/2020/papers/00_vanderwalt/00_vanderwalt.rst). If you don't have one, no worries! You can still publish your paper as is.

**Comment by valeriabarra** [comment]
Thanks @deniederhut! I added them

**Comment by deniederhut** [comment]
Hi @vahtras ! Did you have further feedback on this paper?

**Comment by vahtras** [comment]
I am pleased with the result and I have not more comments

File: papers/libceed-paper/bibliography.bib [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    year = "2020"
+}
+
+@misc{Numba,
+    title = "Numba",
+    url = "https://numba.pydata.org/",
+    year = "2020"
+}
+
+@misc{NumPy,
```
Comment (deniederhut) [inline papers/libceed-paper/bibliography.bib]: Could I ask you to update the citation for NumPy? https://ieeexplore.ieee.org/document/5725236

File: papers/libceed-paper/bibliography.bib [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+            and Dmitry Karpeyev and Dinesh Kaushik and Matthew~G. Knepley and Dave~A. May and Lois Curfman McInnes
+            and Richard Tran Mills and Todd Munson and Karl Rupp and Patrick Sanan
+            and Barry~F. Smith and Stefano Zampini and Hong Zhang and Hong Zhang},
+  title  = {{PETS}c Users Manual},
+  institution = {Argonne National Laboratory},
+  year   = 2020,
+  number = {ANL-95/11 - Revision 3.13}
+}
+
+@misc{ProjectJupyter,
```
Comment (deniederhut) [inline papers/libceed-paper/bibliography.bib]: Could we ask you to update the citation for Jupyter? http://ebooks.iospress.nl/publication/42900

File: papers/libceed-paper/bibliography.bib [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  doi          = {10.5281/zenodo.3860804}
+}
+
+@misc{Nekwebsite,
+    title = "Nek5000",
+    url = "https://nek5000.mcs.anl.gov/",
+    year = "2020"
+}
+
+@misc{Numba,
```
Comment (deniederhut) [inline papers/libceed-paper/bibliography.bib]: Could I ask you to update the citation for numba? https://dl.acm.org/doi/10.1145/2833157.2833162

Comment (valeriabarra) [reply papers/libceed-paper/bibliography.bib]: Sure thing. I updated them. Thanks for pointing out the appropriate references.

**Comment by deniederhut** [comment]
Sweet! I think this is good to go.

**Review by cbcunc — APPROVED** [review]
Passes final proceeding committee walk-through.

**Pull request merged**